tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -113,31 +110,10 @@ public function run(UnifiedTestCase $test): void
 
             throw $e;
         } finally {
-            /* An EntityMap observer should be invoked irrespective of the test
-             * succeeding or failing. Since the callable itself might throw, we
-             * need to ensure doTearDown() will still be called. */
-            try {
-                if (isset($this->entityMapObserver, $this->entityMap)) {
-                    call_user_func($this->entityMapObserver, $this->entityMap);
-                }
-            } finally {
-                $this->doTearDown($hasFailed);
-            }
+            $this->doTearDown($hasFailed);
         }
     }
 
-    /**
-     * Defines a callable to receive the EntityMap after each test.
-     *
-     * This function is primarily used by the Atlas testing workload executor.
-     *
-     * @param callable(EntityMap):void $entityMapObserver
-     */
-    public function setEntityMapObserver(callable $entityMapObserver): void
```

Comment:
This is called from the PHP workload executor in the Atlas testing project (see: [`workload-executor.php`](https://github.com/mongodb-labs/drivers-atlas-testing/blob/8cc57d7efd954d6133dc7afac5a19e85fe43a873/integrations/php/workload-executor.php#L86)).

If you'd like to repurpose this PR you can add a link to the https://github.com/mongodb-labs/drivers-atlas-testing/ repository.

Reply:
Indeed, it was written in the comment above.

I updated the PR to add link and test this feature.